dynamo/tools/clustering.py

```diff
@@ -382,33 +402,23 @@ def cluster_community(
     copy: bool = False,
     **kwargs
 ) -> Union[AnnData, None]:
-    """A base function for detecting communities and inserting results into adata with algorithms specified parameters passed in.
-    Adjacent matrix retrieval priority: adj_matrix > adj_matrix_key > others
+    """A base function for detecting communities and inserting results into adata with algorithms specified parameters
+    passed in. Adjacent matrix retrieval priority: adj_matrix > adj_matrix_key > others
 
-    Parameters
-    ----------
-    adata
-        adata object
-    method
-        community detection method, by default "leiden"
-    result_key
-        the key where the results are stored in obs, by default None
-    adj_matrix
-        adj_matrix used for clustering, by default None
-    adj_matrix_key
-        adj_matrix_key in adata.obsp used for clustering
-    use_weight
-        if using graph weight or not, by default False meaning using connectivities only (0/1 integer values)
-    no_community_label
-        the label value used for nodes not contained in any community, by default -1
-    layer
-        some adata layer which cluster algorithms will work on, by default None
-    cell_subsets
-        cluster only a subset of cells in adata, by default None
-    cluster_and_subsets
-        A tuple of 2 elements (cluster_key, allowed_clusters).filtering cells in adata based on cluster_key in adata.obs and only reserve cells in the allowed clusters, by default None
-    directed
-        if the edges in the graph should be directed, by default False
+    Args:
+        adata: adata object
+        method: community detection method, by default "leiden"
+        result_key: the key where the results are stored in obs, by default None
+        adj_matrix: adj_matrix used for clustering, by default None
+        adj_matrix_key: adj_matrix_key in adata.obsp used for clustering
+        use_weight: if using graph weight or not, by default False meaning using connectivities only (0/1 integer
+            values)
+        no_community_label: the label value used for nodes not contained in any community, by default -1
+        layer: some adata layer which cluster algorithms will work on, by default None
+        cell_subsets: cluster only a subset of cells in adata, by default None
+        cluster_and_subsets: A tuple of 2 elements (cluster_key, allowed_clusters).filtering cells in adata based on
+            cluster_key in adata.obs and only reserve cells in the allowed clusters, by default None
+        directed: if the edges in the graph should be directed, by default False
```

Comment:
Is the default value supposed to be False? It appears to be set as True currently, but I wanted to confirm